### PR TITLE
Add Armenian layout

### PR DESCRIPTION
**Pull request by rafasaurus**
Compiled and tested on an android device.

### CODE REVIEW

File: res/xml/armenian_ph_am.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        <key key0="ռ"/>
+        <key key0="տ"/>
+        <key key0="ը"/>
+        <key key0="ւ"/>
+        <key key0="ի"/>
+        <key key0="օ"/>
+        <key key0="պ"/>
+    </row>
+    <row>
+        <key key0="ա" key1="tab" key2="&quot;"/>
```
Comment (Julow) [inline res/xml/armenian_ph_am.xml:28]: `&quot;` is present twice.

File: res/xml/armenian_ph_am.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        <key key0="յ" key3="+" key2="="/>
+        <key key0="կ" key3="{" key4="}"/>
+        <key key0="լ" key3="[" key4="]"/>
+        <key key0="խ" key3="\" key2="|"/>
+    </row>
+    <row>
+        <key key0="shift" key2="loc capslock"/>
+        <key key0="զ" key1=""/>
+        <key key0="ղ" key1=""/>
+        <key key0="ց" key1=""/>
```
Comment (Julow) [inline res/xml/armenian_ph_am.xml:43]: These `key1` are not doing anything.

File: res/xml/armenian_ph_am.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Armenian" script="armenian">
+    <row>
```
Comment (Julow) [inline res/xml/armenian_ph_am.xml:3]: The layout includes all ASCII punctuations except `\\` and `` ` ``. Is it intended ?

**Review by Julow — COMMENTED** [review]
Here's how it looks:

shot

File: res/xml/armenian_ph_am.xml (hunk 1 above)
Comment (rafasaurus) [reply res/xml/armenian_ph_am.xml:28]: This one will be changed to "`"

File: res/xml/armenian_ph_am.xml (hunk 2 above)
Comment (rafasaurus) [reply res/xml/armenian_ph_am.xml:43]: This will be removed.

File: res/xml/armenian_ph_am.xml (hunk 3 above)
Comment (rafasaurus) [reply res/xml/armenian_ph_am.xml:3]: \\ is included.
` will be added

**Review by Julow — APPROVED** [review]
Perfect, thanks!

**Pull request merged**